controllers/auth.js

```diff
@@ -54,10 +54,28 @@ const refresh = async (req, res) => {
   res.status(200).json(tokens)
 }
 
+const sendResetPasswordEmail = async (req, res) => {
+  const { email } = req.body;
+
+  const resetToken = await authService.sendResetPasswordEmail(email)
+
+  res.status(200).json({ resetToken });
+}
+
+const updatePassword = async (req, res) => {
+  const { password, resetToken } = req.body
+
+  await authService.updatePassword(resetToken, password)
+
+  res.redirect(process.env.CLIENT_URL)
```

Comment:
we don't need to do a redirect because the request will be from the client and we will wait for a response

Reply:
updated to send a 204 request